HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -282,4 +287,7 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     process = customizeHLTfor41632(process)
     process = customizeHLTfor42410(process)
 
+    if menuType in ["GRun","HIon","PIon","PRef"]:
```

Comment:
> On the other hand, we also have other menu "types" which would require this customisation (e.g. "2023v12", see [1] in [2]). Even though these menus are not used in standard tests (PR tests, or IBs), it would be nice not to break them. 

if those are not to be broken, I think a test of some kind is in order.

Reply:
I think the easiest is to add a corresponding "addon" tests (I'll look into it, but I think this PR could proceed in parallel).

Reply:
Well, I suggest to simply drop the test/check on menuType and be done with it.